ApplicationCode/Commands/FractureCommands/RicNewFractureModelFeature.cpp

```diff
@@ -117,48 +117,34 @@ void RicNewFractureModelFeature::setupActionLook( QAction* actionToSetup )
 //--------------------------------------------------------------------------------------------------
 bool RicNewFractureModelFeature::isCommandEnabled()
 {
-    if ( selectedFractureModelCollection() )
-    {
-        return true;
-    }
-
-    return false;
+    return selectedFractureModelCollection() != nullptr;
 }
 
 //--------------------------------------------------------------------------------------------------
 ///
 //--------------------------------------------------------------------------------------------------
 RimFractureModelCollection* RicNewFractureModelFeature::selectedFractureModelCollection()
 {
-    std::vector<caf::PdmUiItem*> allSelectedItems;
-    caf::SelectionManager::instance()->selectedItems( allSelectedItems );
-    if ( allSelectedItems.size() != 1u ) return nullptr;
+    std::vector<caf::PdmUiItem*> selectedItems;
+    caf::SelectionManager::instance()->selectedItems( selectedItems );
+    if ( selectedItems.size() != 1u ) return nullptr;
 
-    caf::PdmUiItem* pdmUiItem = allSelectedItems.front();
+    caf::PdmUiItem* pdmUiItem = selectedItems.front();
 
-    RimFractureModelCollection* objToFind = nullptr;
-
-    caf::PdmObjectHandle* objHandle = dynamic_cast<caf::PdmObjectHandle*>( pdmUiItem );
+    RimFractureModelCollection* perforationCollection = nullptr;
```

Comment:
Rename to fractureModelCollection

Reply:
I guess it is no secret where I stole the code from now! ;-) Will fix.